plugins/primus_test/lisp/memcheck-malloc.lisp

```diff
@@ -3,6 +3,8 @@
 (defmethod call (name ptr)
   (when (and ptr (= name 'free)
              (not (= ptr *malloc-zero-sentinel*)))
+    (unless (memcheck-is-tracked 'malloc ptr)
```

Comment:
For so far unknown to me reason this hack produces false positives. Maybe it is just my inputs, maybe this is something wrong with this idea (by the way, instead of `1` we shall use a parameter here). Anyway, let's merge this into master experiment more with it to see if really a good or bad idea.